### PR TITLE
Cherry-pick --skip-steps from master

### PR DESCRIPTION
**Pull request by jprzychodzen**
Original message

Add flag --skip-steps

This change enables runetime test config change - skipping any number of
steps without a need to modify config file.

/assign @mborsz

### CODE REVIEW

File: clusterloader2/cmd/clusterloader.go [hunk 1]
```diff
@@ -18,6 +18,7 @@ package main
 
 import (
 	"fmt"
+	"k8s.io/perf-tests/clusterloader2/pkg/modifier"
```
Comment (mborsz) [inline clusterloader2/cmd/clusterloader.go:21]: This should go below, but I see it was like that in the origin PR: https://github.com/kubernetes/perf-tests/pull/1247

**Comment by wojtek-t** [comment]
@jprzychodzen - was this cherrypicked to 1.18? If not - please cherrypick, the branches should be more-or-less continuous.

**Comment by jprzychodzen** [comment]
@wojtek-t It was cherrypicked in https://github.com/kubernetes/perf-tests/pull/1256

**Pull request merged**